src/plots/barPlot.ts

```diff
@@ -420,51 +445,65 @@ export module Plots {
         let text = this._labelFormatter(primaryAccessor(d, i, dataset)).toString();
         let w = attrToProjector["width"](d, i, dataset);
         let h = attrToProjector["height"](d, i, dataset);
-        let x = attrToProjector["x"](d, i, dataset);
-        let y = attrToProjector["y"](d, i, dataset);
+        let baselineX = attrToProjector["x"](d, i, dataset);
+        let baselineY = attrToProjector["y"](d, i, dataset);
```

Comment:
Rather than "baseline", let's use "base" to avoid confusion with the actual `baseline` of the `Plots.Bar`.